custom/icds_reports/ucr/data_sources/thr_forms_v2.json

```diff
@@ -51,11 +51,13 @@
         "create_index": true,
         "column_id": "supervisor_id",
         "expression": {
-          "location_id_expression": {
+          "location_id": {
             "type": "named",
             "name": "user_location_id"
           },
-          "type": "location_parent_id"
+          "location_type": "supervisor",
+          "location_propery": "_id",
+          "type": "ancestor_location"
```

Comment:
is this more efficient if there are no other expressions in the data source that also need it?

Reply:
Actually, looks like `ancestor_location` location makes two queries, one to get location and the other to get all ancestors. And `location_parent_id` makes only one query. So, it makes sense to use it if there is only 2 or more indicators needing `location_parent_id`

Reply:
Updated the only two files with single uses to use `location_parent_id`